robottelo/ui/globalparameters.py

```diff
@@ -0,0 +1,40 @@
+# -*- encoding: utf-8 -*-
+"""Implements Global Parameters ."""
+
+from robottelo.ui.base import Base
+from robottelo.ui.locators import common_locators, locators
+from robottelo.ui.navigator import Navigator
+
+
+class GlobalParameters(Base):
+    """Provides the CRUD functionality for Global Parameters."""
+
+    def navigate_to_entity(self):
+        """Navigate to Global Parameters entity page"""
+        Navigator(self.browser).go_to_global_parameters()
+
+    def _search_locator(self):
+        """Specify locator for Global Parameters entity search procedure"""
+        return locators['globalparameters.select']
+
+    def create(self, name, value=None, hidden_value=False):
+        """Creates a Global Parameter"""
+        self.click(locators['globalparameters.new'])
+        self.assign_value(locators['globalparameters.name'], name)
+        if value is not None:
+            self.assign_value(locators['globalparameters.value'], value)
+        self.assign_value(
```

Comment:
better to make `hidden_value=None` and use usual condition. Otherwise you will get to that element all the time. And + you will have same signature as you have in update method